bridgesample/src/main/java/com/livefront/bridgesample/scenario/activity/LargeDataActivity.kt

```diff
@@ -49,14 +49,14 @@ class LargeDataActivity : BridgeBaseActivity() {
         private const val ARGUMENTS_KEY = "arguments"
 
         fun getArguments(
-            activity: LargeDataActivity
-        ): LargeDataActivityArguments? = activity
+                activity: LargeDataActivity
```

Comment:
We should make sure we don't make any unnecessary formatting changes like. At some point I'll drop a formatter into the project to avoid stuff like this but for now just make sure you manually adhere to what the format of the files seem to be.